src/autoscaler/api/broker/broker.go

```diff
@@ -125,12 +125,12 @@ func (b *Broker) Provision(ctx context.Context, instanceID string, details domai
 	}
 
 	if policy != nil {
-		policyGuid, err := uuid.NewV4()
+		policyGuid := uuid.NewString()
 		if err != nil {
 			b.logger.Error("get-default-policy-create-guid", err, lager.Data{"instanceID": instanceID})
 			return result, apiresponses.NewFailureResponse(errors.New("error generating policy guid"), http.StatusInternalServerError, "get-default-policy-create-guid")
 		}
```

Comment:
Nice!

```suggestion
```

Reply:
thanks for that catch! that slipped somehow through during the adjustment, fixed with 0e86c24